doc/en/components/grids/_shared/column-hiding.md

```diff
@@ -73,17 +73,17 @@ Let's start by creating our `{ComponentName}` and binding it to our data. We wil
 ```
 
 ```tsx
-<IgrGrid autoGenerate="false" width="100%" height="560px" allowFiltering="true">
-    <IgrColumn field="ID" dataType="string" sortable="true" hidden="true"></IgrColumn>
-    <IgrColumn field="ContactName" dataType="String" sortable="true" hidden="true"></IgrColumn>
-    <IgrColumn field="ContactTitle" dataType="string" sortable="true"></IgrColumn>
-    <IgrColumn field="City" dataType="string" sortable="true"></IgrColumn>
-    <IgrColumn field="CompanyName" dataType="string" sortable="true"></IgrColumn>
-    <IgrColumn field="Fax" dataType="string" sortable="true"></IgrColumn>
-    <IgrColumn field="Address" dataType="string" sortable="true"></IgrColumn>
-    <IgrColumn field="PostalCode" dataType="string" sortable="true"></IgrColumn>
-    <IgrColumn field="Country" dataType="string" sortable="true"></IgrColumn>
-    <IgrColumn field="Phone" dataType="string" sortable="true"></IgrColumn>
+<IgrGrid autoGenerate={false} width="100%" height="560px" allowFiltering={true}>
+    <IgrColumn field="ID" dataType="string" sortable={true} hidden={true}></IgrColumn>
+    <IgrColumn field="ContactName" dataType="String" sortable={true} hidden={true}></IgrColumn>
```

Comment:
Everywhere `dataType` values should start with lowercase letter `dataType="String"` becomes `dataType="string"`